<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1615460870827' stop='1615460890520'>
    <name>Check sign in</name>
    <title>Check sign in</title>
    <test-cases>
        <test-case start='1615460870828' status='failed' stop='1615460890513'>
            <name>should check Yandex music</name>
            <title>should check Yandex music</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>NoSuchElementError: No element found using locator: By(css selector, .typo-h1)</message>
                <stack-trace>NoSuchElementError: No element found using locator: By(css selector, .typo-h1)
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\built\element.js:814:27
                    at ManagedPromise.invokeCallback_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1376:14)
                    at TaskQueue.execute_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\User\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7
                    at processTicksAndRejections (internal/process/task_queues.js:93:5)Error
                    at ElementArrayFinder.applyAction_ (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\built\element.js:459:27)
                    at ElementArrayFinder.&lt;computed> [as getText] (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\built\element.js:91:29)
                    at ElementFinder.&lt;computed> [as getText] (C:\Users\User\AppData\Roaming\npm\node_modules\protractor\built\element.js:831:22)
                    at UserContext.&lt;anonymous> (C:\Users\User\Protractor_project\secondPull\test\test.spec.js:146:23)
                    at processTicksAndRejections (internal/process/task_queues.js:93:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1615460890514' status='pending' stop='1615460890516'>
            <name>should compare two phones</name>
            <title>should compare two phones</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1615460890516' status='pending' stop='1615460890517'>
            <name>should delete items from the comparison list</name>
            <title>should delete items from the comparison list</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1615460890517' status='pending' stop='1615460890518'>
            <name>should sort cameras by price</name>
            <title>should sort cameras by price</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
        <test-case start='1615460890518' status='pending' stop='1615460890519'>
            <name>should sort by tag</name>
            <title>should sort by tag</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>This test was ignored</message>
                <stack-trace></stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>